test.js

```diff
@@ -33,6 +33,35 @@ test('efficiency', (t) => {
     }, 0)
   })
 
+  t.test('flanking whitespace', (t) => {
+    const timeoutId = setTimeout(() => {
+      t.fail('did not pass in 10ms')
+    }, 10)
+
+    t.deepEqual(trimLines(whitespace + '\na\n' + whitespace), '\na\n')
+
+    setTimeout(() => {
+      clearTimeout(timeoutId)
+      t.end()
+    }, 0)
+  })
+
+  t.test('internalized whitespace ', (t) => {
+    const timeoutId = setTimeout(() => {
+      t.fail('did not pass in 30ms')
+    }, 30)
```

Comment:
Why is this 30 instead of 10?
Should all be 30, and mean “fast enough”?

Reply:
No, this test is slower because it's the edge case that originally caused problems. It might pass in 10s with the fully non-regex version though I'd have to try it.

Reply:
Actually on second test, it appears it does pass in 10 ms, I must have accidentally tested 1ms or something.

Reply:
On a third revision it appears they all need 20-30ms, they are intermittently failing with 10. Good call.

Reply:
Also, we’re not testing `xms`. We’re testing: fast enough. Compared to alternative ways which result in like a second or more.
Timeouts mainly, but any timing tests in general are really hard to get right.
Older Node versions or Windows for example will make everything slightly slower. Because these numbers are really small, they will leed to randomly breaking CIs.